### ISSUE TITLE
Reading Contact - Not working

### ISSUE THREAD

**Issue by sundaravelit** [cfjedimaster/Cordova-Examples#7]
Hi,
I just copy paste steal contact file to my phonegap, but it's not working...

**Comment by cfjedimaster** [comment]
How is it not working? Provide details about the failure.

**Comment by sundaravelit** [comment]
Thank you for Reply!

i compiled, and i got apk file & installed on my phone
but
it shows only white screen..

**Comment by cfjedimaster** [comment]
Did you install the plugins as described in the readme?
When you use remote debugging, what error do you see?

**Comment by sundaravelit** [comment]
yes i installed plugin using "cordova plugin add org.apache.cordova.contacts"

i didn't try remote debugging ..

**Comment by sundaravelit** [comment]
am i need to change anything on my config.xml ?

**Comment by cfjedimaster** [comment]
You shouldn't need to worry about config.xml. Please try remote debugging as it should show you an error in the console.

**Comment by sundaravelit** [comment]
I'm new to phonegap, 

can u suggest me how to do that? please..

**Comment by cfjedimaster** [comment]
Go to my blog (raymondcamden.com), to the About page, and see my list of articles. I have two that go into detail about how to do remote debugging.

**Comment by cfjedimaster** [comment]
Also see https://www.youtube.com/watch?v=IouRD05fqmo&list=UU8KROrnEHSnnV3z5J_FoSIg

**Comment by sundaravelit** [comment]
thank you so much for your reply!

I'll check with that..

can u suggest me some sites or books for phonegap beginners to advance level..?

and one more doubt. why phonegap is slow and how to solve that??

**Comment by cfjedimaster** [comment]
Um, well, I blog on PG a lot, you should go to phonegap.com and check the
blog as they aggregate active PhoneGap bloggers like myself.

As for performance, I'd need a lot more info to help you and that isn't
really appropriate for a GitHub issue.

On Fri, Jan 2, 2015 at 12:09 PM, Sundaravel <notifications@github.com>
wrote:

> thank you so much for your reply!
>
> I'll check with that..
>
> can u suggest me some sites or books for phonegap beginners to advance
> level..?
>
> and one more doubt. why phonegap is slow and how to solve that??
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfjedimaster/Cordova-Examples/issues/7#issuecomment-68547691>
> .
>



-- 
===========================================================================
Raymond Camden, Web Standards Evangelist

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden
